CHANGELOG.md

```diff
@@ -1,18 +1,88 @@
 # Changelog
 
-## 1.1.0
-**Features:**
+## 2.0.0
 
-- **Add support for WebSockets!**
-- `WSocket` class is now available when you import the `w_transport` package.
+> The 2.0.0 release is a major breaking release. While many of the patterns from
+> 1.0.x were maintained, the HTTP API was broken up into several request classes
+> and two response classes for a much more robust and useful API. As such, there
+> is no backwards compatibility, but a migration guide is included below.
+
+### Features
+
+- **WebSockets**
+  - Single API for the browser and the Dart VM.
+  - Option to use SockJS library in place of native WebSockets for the ability
+    to fall back to XHR streaming (configuration only, no API usage difference).
+
+- **HTTP**
+  - Support for most commonly used request types:
+    - `Request` (content-type: text/plain)
+    - `JsonRequest` (content-type: application/json)
+    - `FormRequest` (content-type: application/x-www-form-urlencoded)
+    - `MultipartRequest` (content-type: multipart/form-data)
+  - Synchronous access to response bodies as bytes, text, and JSON.
+  - Asynchronous request bodies (`StreamedRequest`).
+  - Asynchronous response bodies via `streamGet()`, `streamPost()`, etc. on any
+    of the above request classes.
+  - Automatic request encoding and response decoding.
+
+- **Mocks**
+  - Because this library is designed such that as much code as possible is
```

Comment:
`Because this library is designed to be platform-agnostic, it's easy to introduce mocks by treating tests as another platform,...`